app/src/pages/eventspage/EventsEditPage.tsx

```diff
@@ -40,6 +40,18 @@ import { getUnixTime } from 'date-fns';
     
     const [eventLocal, setEventLocal] = useState<any>();
 
+    const [imageQuery, setImageQuery] = useState<string>();
+
+    const imageOptions = new Map()
+    imageOptions.set("company", ["https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/M%C3%BCnster%2C_LVM%2C_B%C3%BCrogeb%C3%A4ude_--_2013_--_5149-51.jpg/1200px-M%C3%BCnster%2C_LVM%2C_B%C3%BCrogeb%C3%A4ude_--_2013_--_5149-51.jpg"]);
+    imageOptions.set("google", ["https://s39939.pcdn.co/wp-content/uploads/2023/02/iStock-1169427542.jpg",
+                                "https://storage.googleapis.com/gweb-uniblog-publish-prod/images/PxG_GVE_Blog_Header-bike_1.width-1300.png",
+                                "https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/Googleplex_HQ_%28cropped%29.jpg/1200px-Googleplex_HQ_%28cropped%29.jpg",
+                                "https://s39939.pcdn.co/wp-content/uploads/2023/02/iStock-1169427542.jpg",
+                                "https://storage.googleapis.com/gweb-uniblog-publish-prod/images/PxG_GVE_Blog_Header-bike_1.width-1300.png",
+                                "https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/Googleplex_HQ_%28cropped%29.jpg/1200px-Googleplex_HQ_%28cropped%29.jpg"
+                                ]);
+    imageOptions.set("meta",    ["https://about.fb.com/wp-content/uploads/2021/10/Meta-Planets-img-16x9-1.jpg?w=1200"]);
```

Comment:
similar to prev PR, these can be put on a file (maybe a json) and then be read from there